javascript-arrays/main.js

```diff
@@ -0,0 +1,19 @@
+const colors = ['red', ' white', ' blue'];
+console.log('value of colors[0]: ', colors[0]);
+console.log('value of colors[1]: ', colors[1]);
+console.log('value of colors[2]: ', colors[2]);
+
+console.log('America is ' + colors + '.');
```

Comment:
Instead of concatenating the whole `colors` array, you should be concatenating the individual values of the array. The whole string should be "America is red white and blue". So you need to concatenate a string that includes the spaces and the `and`.